wkcuber/downsampling.py

```diff
@@ -109,8 +107,15 @@ def create_parser():
         action="store_true",
     )
 
+    parser.add_argument(
```

Comment:
I think there are two other occurrences of this code. Can you extract it similar to how `add_isotropic_flag` is defined in the utils?